python/transformations/Hugging-Face-Model/hugging_face_model.py

```diff
@@ -22,16 +22,16 @@ def on_event_data_handler(self, data: EventData):
         label = prediction['label']
         score = prediction['score']
 
-        # Write df to output stream
-        self.output_stream.parameters.buffer \
+        # Publish df to output stream
+        self.stream_producer.timeseries.buffer \
             .add_timestamp_nanoseconds(data.timestamp_nanoseconds) \
-            .add_tags(data.tags) \
             .add_value("label", label) \
             .add_value("sentiment", score) \
-            .write()
+            .add_tags(data.tags) \
+            .publish()
 
-    # Callback triggered for each new table data.
-    def on_parameter_data_handler(self, data: ParameterData):
+    # triggered for each new timeseries data.
+    def on_parameter_data_handler(self, stream_consumer: qx.StreamConsumer, data: qx.TimeseriesData):
```

Comment:
Now that we've moved from parameter data to time-series data, should this function name be changed too?